src/managers/kafka.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Supporting objects for Kafka utils and management."""
+
+from charms.kafka.v0.client import KafkaClient
+
+from core.cluster import ClusterContext
+from core.workload import WorkloadBase
+from literals import KAFKA_TOPIC
+
+
+class KafkaManager:
+    """Object for handling Kafka."""
+
+    def __init__(self, context: ClusterContext, workload: WorkloadBase) -> None:
+        self.context = context
+        self.workload = workload
+
+    def brokers_active(self) -> bool:
+        """Check that Kafka is active."""
+        client = KafkaClient(
```

Comment:
**question** It seems that in VM we have a comment here:

```
# FIXME If SSL connection is enabled we need SSL paths added to this KafkaClient
```

Why is not here?

Reply:
This was fixed already, and I did a pass on `TODO`s and `FIXME`s that removed this one. Will update.